README.md

```diff
@@ -13,42 +11,67 @@ The following classes are needed to configure, in order to execute the module.
 ### DynamicAssurance
 #### Configuration
 The following authproc filter configuration options are supported:
-  * `entitlementWhitelist`: Required, an array of strings containing values that can get higher assurance.
-  * `attribute`: Optional, a string that defines the name of the attribute that will store the value of LoA. Defaults to `eduPersonAssurance`.
-  * `candidates`: Optional, an array of strings that contains the allowed LoA values.
-  * `defaultAssurance`: Optional, a string to use as the default value of LoA. Defaults to `https://www.example.org/low`.
-  * `defaultElevatedAssurance`: Optional, a string to use as the default value of LoA. Defaults to `https://www.example.org/Substantial`.
-  * `idpPolicies`: Optional, an array of stings that contains the allowed IdP Policies.
-  * `idpTags`: Optional, an array of stings that contains the allowed IdP Tags.
+  * `attribute`: _Optional_, a string that defines the name of the attribute that will store the value of LoA. Defaults to `eduPersonAssurance`.
+  * `assuranceMap`: _Optional_, an array of
+    * `key`, `array` pairs, that contains the mapping of an attribute value to Assurance values. Defaults to
+    * `strings`, that each contains a regular expression(regex) pattern of the Assurance values to pass through
+  ```php
+      $assuranceMap = array(
+        'eduPersonAssurance' => array(
+            '1.2.840.113612.5.2.2.1' => array(                  // Classic
+                'https://refeds.org/assurance/IAP/low',
+                'https://refeds.org/assurance/IAP/medium',
+            ),
+            '1.2.840.113612.5.2.2.5' => array(                  // MICS
+                'https://refeds.org/assurance/IAP/low',
+                'https://refeds.org/assurance/IAP/medium',
+            ),
+        ),
+        '/^https:\/\/refeds\.org\/assurance/m',                 // REFEDS passthrough values
+        '/^https:\/\/aarc-community\.org\/assurance/m',         // AARC passthrough values
+    );
+  ```
+  :warning: The configured Assurance Map array is merged recursively with the default one from above.
+  * `defaultAssurance`: _Optional_, a string to use as the default value of LoA. Defaults to `https://www.example.org/low`.
```

Comment:
Should default to null, i.e. no assurance value is assigned by default

Reply:
Done